docs/index.md

```diff
@@ -3,12 +3,19 @@
 page_title: "envbuilder Provider"
 subcategory: ""
 description: |-
-  
+  Envbuilder Provider
+  The Envbuilder provider can be used to check for the presence of a Docker image previously built by Envbuilder https://github.com/coder/envbuilder.
+  This allows re-using a previously built image pushed to a Docker registry without having to rebuild it.
+  An image built by Envbuilder also embeds the Envbuilder binary inside the image, allowing for the image to be used to build other images.
 ---
 
 # envbuilder Provider
 
+# Envbuilder Provider
 
+The Envbuilder provider can be used to check for the presence of a Docker image previously built by [Envbuilder](https://github.com/coder/envbuilder).
+This allows re-using a previously built image pushed to a Docker registry without having to rebuild it.
+An image built by Envbuilder also embeds the Envbuilder binary inside the image, allowing for the image to be used to build other images.
```

Comment:
This is a bit of a fringe case, do we want to mention it, might seem like a feature and maybe it is, but we'd be making it official? 😄

Reply:
Fair :+1: